src/lib/CMakeLists.txt

```diff
@@ -46,6 +46,19 @@ set_target_properties(partio PROPERTIES
     POSITION_INDEPENDENT_CODE ON
     SOVERSION ${PARTIO_VERSION_MAJOR})
 
+set(TBB_INCLUDE_DIRS "$ENV{TBB_ROOT}/include")
+set(CMAKE_MODULE_PATH "$ENV{OpenVDB_ROOT}/lib/cmake/OpenVDB"
+    "${OpenVDB_ROOT}/lib/cmake/OpenVDB"
+    ${CMAKE_MODULE_PATH})
```

Comment:
`CMAKE_MODULE_PATH` usually shouldn't need to be modified. If you set `CMAKE_PREFIX_PATH` to include the `OpenVDB_ROOT` then in theory `find_package(OpenVDB)` should work without needing to special-case openvdb here.